libraries/node-core-library/src/Terminal/Terminal.ts

```diff
@@ -0,0 +1,142 @@
+import * as colors from 'colors';
```

Comment:
> * as col [](start = 6, length = 9)

Missing copyright banner.  There is a tslint rule.,  We should enable it.

Reply:
We'll do that in a separate PR.

---
In reply to: [227575011](https://github.com/Microsoft/web-build-tools/pull/898#discussion_r227575011) [](ancestors = 227575011)